metagov/metagov/core/views.py

```diff
@@ -274,13 +274,15 @@ def plugin_auth_callback(request, plugin_name):
     plugin_views = importlib.import_module(f"metagov.plugins.{plugin_name}.views")
 
     try:
-        return plugin_views.auth_callback(
+        response = plugin_views.auth_callback(
             type=type,
             code=code,
             redirect_uri=redirect_uri,
             community=community,
             state=state_to_pass,
+            request=request
         )
+        return response if response else HttpResponseRedirect(redirect_uri)
```

Comment:
We need to pass through the `state` if present. this will be the original state that was passed from the Driver to the authorize endpoint (it's not documented [here](https://metagov.policykit.org/redoc/#operation/Authorize%20plugin), but should be!). We just need to pass that `state` back on completion so that the Driver can validate it.

```suggestion
        return response if response else redirect_with_params(redirect_uri, {"state": state_to_pass})
```